cla_public/templates/accessibility-statement.html

```diff
@@ -19,43 +20,51 @@ <h2 class="govuk-heading-m">{{ _('Using this service') }}</h2>
       {% trans %}change colours, contrast levels and fonts{% endtrans %}
     </li>
     <li>
-      {% trans %}zoom in up to 300% without the text spilling off the screen{% endtrans %}
+      {% trans %}zoom in up to 300&#37; without the text spilling off the screen{% endtrans %}
     </li>
     <li>
       {% trans %}navigate the service using just a keyboard{% endtrans %}
     </li>
   </ul>
   
   <p class="govuk-body">
-    {% trans %}<a class="govuk-link" href="https://mcmw.abilitynet.org.uk/" rel="external">AbilityNet has advice on making your device easier to use if you have a disability.</a>{% endtrans %}
+    <a class="govuk-link" href="https://mcmw.abilitynet.org.uk/" rel="external">{% trans %}AbilityNet has advice on making your device easier to use if you have a disability.{% endtrans %}</a>
   </p>
 
-  <h2 class="govuk-heading-m">{{ _('What to do if you can’t access part of this service') }}</h2>
+  <h2 class="govuk-heading-m">{{ _('Feedback and contact information') }}</h2>
 
   <p class="govuk-body">
-    {% trans %}If you need information on this service in a different format, you can contact us at <a class="govuk-link email" href="mailto:civil-legal-advice@digital.justice.gov.uk">civil-legal-advice@digital.justice.gov.uk</a>.{% endtrans %}
+    {% trans
+      email = Element.link_same_window('mailto:civil-legal-advice@digital.justice.gov.uk', 'civil-legal-advice@digital.justice.gov.uk', **{'class': 'email'})
```

Comment:
Does the value for the class key need to be 'govuk-link-email'?

Reply:
No, `govuk-link` and `email` are two different classes.